doc/classes/Vector2.xml

```diff
@@ -112,6 +112,15 @@
 				Cubicly interpolates between this vector and [code]b[/code] using [code]pre_a[/code] and [code]post_b[/code] as handles, and returns the result at position [code]t[/code]. [code]t[/code] is in the range of [code]0.0 - 1.0[/code], representing the amount of interpolation.
 			</description>
 		</method>
+		<method name="direction_to">
+			<return type="Vector2">
+			</return>
+			<argument index="0" name="b" type="Vector2">
+			</argument>
+			<description>
+				Returns the normalized vector directed from current vector to [code]b[/code].
```

Comment:
"directed" doesn't quite sound right. I would say "pointing" here: "pointing from this vector..."

Reply:
Fixed